app/controllers/PayPalOneOff.scala

```diff
@@ -0,0 +1,73 @@
+
+package controllers
+
+import actions.CustomActionBuilders
+import assets.AssetsResolver
+import cats.implicits._
+import com.gu.identity.play.IdUser
+import monitoring.SafeLogger
+import monitoring.SafeLogger._
+import play.api.libs.circe.Circe
+import play.api.libs.json.Json
+import play.api.mvc._
+import services.PaymentAPIService.Email
+import services.{IdentityService, PaymentAPIService, TestUserService}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class PayPalOneOff(
+    actionBuilders: CustomActionBuilders,
+    assets: AssetsResolver,
+    testUsers: TestUserService,
+    components: ControllerComponents,
+    paymentAPIService: PaymentAPIService,
+    identityService: IdentityService
+)(implicit val ec: ExecutionContext) extends AbstractController(components) with Circe {
+
+  import actionBuilders._
+
+  implicit val assetsResolver = assets
+
+  def resultFromEmailOption(email: Option[Email]): Result = {
+    val redirect = Redirect("/contribute/one-off/thankyou")
+    email.fold(redirect)(e => {
```

Comment:
Our convention is to put the `{` before the parameter so:
`email.fold(redirect)({e => `

Reply:
Fixed in: 71d5d7b